### DISCUSSION TITLE
F55E2350-0051-44BA-8471-D7F68B1E1300

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#153] [General]
# F55E2350-0051-44BA-8471-D7F68B1E1300

Coruscant is a city-covered planet that has became the capital of both the Old Republic and the Galactic Empire for many millenia. It is located within the Core Worlds region of the Galaxy, and is believed to be humanity's own planet of origin

https://celestia.mobi/addon?item=F55E2350-0051-44BA-8471-D7F68B1E1300

**Comment by exocritic5457** [comment]
I'm just wondering, but will textures of other planets in the same system be included in the addons, in the event that they're ever made?

(On a side note, Prime's a G-type instead of a B-type, and Platoril is missing. can't wait for more SW systems' updates in the future !)

**Reply by DaveBowman2001** [reply]
It depends on the add-on developers themselves if they want to customize the other planets as well. So far, almost every Star Wars planet add-on here was recovered from older add-ons from 2015 and older, and none really attempted to customize worlds other than the main planets in the system since they placed it on a single star before the [Star Wars Galaxy](https://celestia.mobi/resources/item?item=000CB09B-CB61-4303-AC0C-B0C5BECE6B9E) was released just yesterday

**Reply by DaveBowman2001** [reply]
> (On a side note, Prime's a G-type instead of a B-type, and Platoril is missing. can't wait for more SW systems' updates in the future !)

Just checked, Platoril was missing because of a coding error in the default catalog, should be fixed now. Also, I might misunderstood this but Coruscant Prime is always been a G-type star.....or should it be a hotter B-type one?

**Reply by exocritic5457** [reply]
It appears to be listed as a B-type in both Canon and Legends, and Coruscant having a 365 day year at its average distance of 229 million km is consistent with the mass of a B-type star

**Reply by DaveBowman2001** [reply]
Ah I see. Though making it also a B-type star wouldn't really make any sense imho, since they are hotter, the HZ should be much further away (ruling out a supposed 365-day year for Coruscant). In addition, the yellowish [sunrise scene in the very opening of ROTS](https://makeagif.com/amp/kqnq8N) is consistent with a K/G-type star instead

**Reply by exocritic5457** [reply]
That's valid. Vandor-3 would be its own problem at that

**Reply by exocritic5457** [reply]
Makes me wonder what sci-fi space magic they pulled for life, let alone humans to evolve around a purported B-type main-sequence star, a class that dies off in the cosmic blink of an eye
